assets/src/components/customers/CustomersTable.tsx

```diff
@@ -77,8 +77,10 @@ const CustomersTable = ({
       dataIndex: 'last_seen_at',
       key: 'last_seen_at',
       render: (value: string, record: Customer) => {
-        const {id, pathname, current_url, last_seen} = record;
-        const formatted = dayjs.utc(value || last_seen).format('MMMM DD, YYYY');
+        const {id, pathname, current_url, last_seen_at} = record;
+        const formatted = dayjs
+          .utc(value || last_seen_at)
```

Comment:
`value` and `last_seen_at` should be the same in this case, so we can just get rid of the `|| last_seen_at` here i believe